app/assets/stylesheets/alchemy/frame.scss

```diff
@@ -273,7 +273,7 @@ div#user_info {
   }
 }
 
-#subnav_additions {
+#locked_pages {
```

Comment:
Avoid using id selectors<br>Selector `locked_pages` should be written in lowercase with hyphens